src/openrct2/Game.cpp

```diff
@@ -736,6 +736,15 @@ static void game_load_or_quit_no_save_prompt_callback(int32_t result, const utf8
     }
 }
 
+static void game_new_game_window_callback(const utf8* path)
```

Comment:
```suggestion
static void NewGameWindowCallback(const utf8* path)
```

We use TitleCase for functions (I know it is inconsistent here)